pkg/lib/configreader/interface_test.go

```diff
@@ -14,15 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package interfaces_test
+package configreader
```

Comment:
We currently have the convention of using the external package for tests, but I'm on board for switching to internal packages going forward since it simplifies some things, and [Dave Cheney is ok with it](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_prefer_internal_tests_to_external_tests) :)